frontend-thea-aa/src/lib/components/ContaminatedSites.svelte

```diff
@@ -7,25 +7,28 @@
 	let amountToShow = $state(3)
 </script>
 
-<h3>Contaminated Sites</h3>
+<div class="max-w-4xl mx-auto py-6">
+    <h3 class="text-l font-bold mb-4 text-gray-800">Contaminated Sites Update (Superfund)</h3>
 
-<table>
-	<thead>
-		<tr>
-			<th>Status</th>
-			<th>Site</th>
-			<th>Hazard Score</th>
-		</tr>
-	</thead>
-	<tbody>
-		{#each props.items.slice(0, amountToShow) as item}
-			<tr>
-				<td>{item.status}</td>
-				<td>{item.name}</td>
-				<td>{item.hazardScore}</td>
-			</tr>
-		{/each}
-	</tbody>
-</table>
-
-<LoadMore onclick={() => (amountToShow += 3)} />
+    <table class="min-w-full table-auto border-collapse">
+        <thead>
+            <tr class="bg-gray-100">
+                <th class="px-4 py-2 text-left text-sm font-medium text-gray-700">Status</th>
```

Comment:
same as previous comments with regards to specific colors https://www.shadcn-svelte.com/docs/theming